app/src/main/java/com/android/partagix/ui/screens/ViewAccount.kt

```diff
@@ -0,0 +1,183 @@
+                          focusedIndicatorColor = Color.Gray,
+                          disabledIndicatorColor = Color.Gray,
+                          unfocusedIndicatorColor = Color.Gray,
+                          focusedContainerColor = Color.Transparent,
+                          unfocusedContainerColor = Color.Transparent,
+                          disabledContainerColor = Color.Transparent),
+                  modifier = Modifier.fillMaxWidth().padding(8.dp).testTag("location"),
+                  readOnly = true,
+                  leadingIcon = { Icon(Icons.Default.LocationOn, contentDescription = null) })
+              val rank = user.rank
+              val stars: String
+              if (rank == "") {
+                stars = "No trust yet"
+              } else {
+                val rating = round(rank.toFloat() * 100) / 100
+                // val rating = 4.5
+                val roundedRating = round(rating).toInt()
+                stars =
+                    when (roundedRating) {
+                      0 -> {
+                        "☆☆☆☆☆ ($rating/5)"
+                      }
+                      1 -> {
+                        "★☆☆☆☆ ($rating/5)"
+                      }
+                      2 -> {
+                        "★★☆☆☆ ($rating/5)"
+                      }
+                      3 -> {
+                        "★★★☆☆ ($rating/5)"
+                      }
+                      4 -> {
+                        "★★★★☆ ($rating/5)"
+                      }
+                      5 -> {
+                        "★★★★★ ($rating/5)"
+                      }
+                      else -> {
+                        "..."
+                      }
+                    }
+              }
+              TextField(
+                  value = stars,
+                  onValueChange = {},
+                  label = { Text("Trust") },
+                  colors =
+                      TextFieldDefaults.colors(
+                          focusedIndicatorColor = Color.Gray,
+                          disabledIndicatorColor = Color.Gray,
+                          unfocusedIndicatorColor = Color.Gray,
+                          focusedContainerColor = Color.Transparent,
+                          unfocusedContainerColor = Color.Transparent,
+                          disabledContainerColor = Color.Transparent),
+                  modifier = Modifier.fillMaxWidth().padding(8.dp).testTag("rating"),
+                  readOnly = true,
+                  leadingIcon = { Icon(Icons.Default.CheckCircle, contentDescription = null) })
+              Spacer(modifier = Modifier.height(16.dp))
+              Row(modifier = Modifier.fillMaxWidth().padding(8.dp, 0.dp).testTag("actionButtons")) {
+                Button(
+                    onClick = { navigationActions.navigateTo(Route.INVENTORY) },
+                    modifier = Modifier.fillMaxWidth(0.5f).testTag("inventoryButton")) {
```

Comment:
0.5f could be a constant

Reply:
Changed the way the UI displays the 2 buttons to be more simple.

Reply:
makes more sense like this !